hack/e2e.sh

```diff
@@ -14,6 +14,21 @@ if [[ -z "$TIMEOUT" ]]; then
     TIMEOUT=20m
 fi
 
-go test ./test/e2e -timeout "$TIMEOUT" -test.v -ginkgo.v "${FOCUS:-}" -ginkgo.noColor -tags e2e "${ARTIFACT_FLAG:-}"
+# start the fake rp server if needed
```

Comment:
Doing this here is wrong since this script already runs in CI alongside a running fake RP?

Reply:
I guess the script will not fail even though 8080 is already bound to a different process because you run it aysnchronously here?

Reply:
I think this is fine, we probably want to remove https://github.com/openshift/release/blob/dcf126e623dae74f034189b6177901a518d23877/ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml#L232

Reply:
:+1: